modules/docusaurus-website/src/get-default-config.ts

```diff
@@ -263,7 +273,8 @@ export function getDocusaurusConfig(config: OcularWebsiteConfig): Config {
               ]
             }
           ],
-          copyright: `Copyright © ${new Date().getFullYear()} OpenJS Foundation`
+          copyright:
+            '<p>Copyright <a href="https://openjsf.org">OpenJS Foundation</a> and vis.gl contributors. All rights reserved. The <a href="https://openjsf.org">OpenJS Foundation</a> has registered trademarks and uses trademarks. For a list of trademarks of the <a href="https://openjsf.org">OpenJS Foundation</a>, please see our <a href="https://trademark-policy.openjsf.org">Trademark Policy</a> and <a href="https://trademark-list.openjsf.org">Trademark List</a>. Trademarks and logos not indicated on the <a href="https://trademark-list.openjsf.org">list of OpenJS Foundation trademarks</a> are trademarks&trade; or registered&reg; trademarks of their respective holders. Use of them does not imply any affiliation with or endorsement by them.</p><p><a href="https://openjsf.org">The OpenJS Foundation</a> | <a href="https://terms-of-use.openjsf.org">Terms of Use</a> | <a href="https://privacy-policy.openjsf.org">Privacy Policy</a> | <a href="https://bylaws.openjsf.org">Bylaws</a> | <a href="https://code-of-conduct.openjsf.org">Code of Conduct</a> | <a href="https://trademark-policy.openjsf.org">Trademark Policy</a> | <a href="https://trademark-list.openjsf.org">Trademark List</a> | <a href="https://www.linuxfoundation.org/cookies">Cookie Policy</a></p>'
```

Comment:
This is quite a wall of text compared to what we had before. Checking at random, https://nodejs.org/en just links to OpenJS

<img width="1103" height="121" alt="Screenshot 2025-08-26 at 14 56 53" src="https://github.com/user-attachments/assets/a9d337dc-95bb-4159-b6ef-0bc63934abfd" />

Reply:
It's copy/pasted from their spec

https://github.com/openjs-foundation/artwork?tab=readme-ov-file#copyright-notices-for-project-website-footers

Reply:
Yeah we can do 12pt or something. cosmos.gl and [GeoDa](https://geodaai.github.io/) have this footer too

Reply:
0.8 rem looks nice and small https://github.com/geodaai/geodaai.github.io/blob/main/website/src/css/custom.css#L69-L73